qiskit/ignis/examples/discriminator.ipynb

```diff
@@ -0,0 +1,288 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import random\n",
+    "import numpy as np\n",
+    "from matplotlib import pyplot as plt\n",
+    "\n",
+    "%matplotlib inline\n",
+    "plt.rcParams['font.size'] = 24\n",
+    "\n",
+    "import qiskit\n",
+    "from qiskit import Aer\n",
+    "from qiskit.ignis.measurement.discriminator.linear import LinearIQDiscriminationFitter\n",
+    "from qiskit.ignis.mitigation.measurement import circuits\n",
+    "from qiskit.result.models import ExperimentResultData"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Functions to make up some fake level 1 data for the qubits\n",
+    "# This could be replaced by Aer simulation once level 1 simulation\n",
+    "# Is available.\n",
+    "def qubit_shot(i0, q0, std):\n",
+    "    return [i0 + random.gauss(0, std), q0 + random.gauss(0, std)]\n",
+    "\n",
+    "def create_shots(i0, q0, shots, qubits):\n",
```

Comment:
Could this just return the `ExperimentResultData` result object for now, until the simulator supports measurement level 1? This would avoid the conversion steps below which could be confusing.

Reply:
Also given you likely need such a result generation function to write tests could you reuse it by just importing this function in the example?